user_mail_server/res_users.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015- Vertel AB (<http://www.vertel.se>).
+#
+#    This progrupdateam is free software: you can redistribute it and/or modify
```

Comment:
progrupdateam?

Reply:
oh, that was copied from a module. I did not see that. It can be removed.